dashboard_data/SpatialAggregation.md

```diff
@@ -0,0 +1,71 @@
+# Aggregate by Spatial Relationship
+
+This expression aggregates data from one layer using a spatial relationship with data from a second layer.
+
+In this specific example, point features ([Global Power Plants](https://www.arcgis.com/home/item.html?id=848d61af726f40d890219042253bedd7)) are being aggregated by polygons ([Time Zones](https://www.arcgis.com/home/item.html?id=312cebfea2624e108e234220b04460b8)) using the **Contains** function.
+
+```javascript
```

Comment:
should be js, right?  maybe does not matter

Reply:
ah, it probably should